evm/interchainTokenFactory.js

```diff
@@ -265,7 +265,7 @@ async function processCommand(_axelar, chain, chains, options) {
 
```

Comment:
trusted address set/remove is still using the new methods, switch those to use `setTrustedAddress(chain, 'hub')`?